src/controls/mod.rs

```diff
@@ -223,30 +226,43 @@ impl Default for ControlState {
             },
             strain_threshold_control: StrainThresholdState {
                 nuance: 0.0,
-                strain_threshold: 0.0,
+                strain_limits: (0.0, 1.0),
             },
             gravity_control: GravityState {
                 nuance: 0.0,
                 min_gravity: 1e-8,
                 max_gravity: 5e-7,
             },
+            show_strain: false,
             frame_rate: 0.0,
             action_queue: RefCell::new(Vec::new()),
         }
     }
 }
 
+pub struct StrainView {
+    pub threshold: f32,
+    pub material: usize,
```

Comment:
Do you plan on varying the material in the future?

Reply:
Not sure, there's only one special one now.